docker/Dockerfile

```diff
@@ -0,0 +1,43 @@
+FROM debian:jessie
+
+ARG DEBIAN_FRONTEND="noninteractive"
+
+ENV LANG C.UTF-8
+
+RUN apt-get update && apt-get install -y \
+    apt-utils
+
+RUN apt-get install -y \
+  git-core \
+  python-pip \
+  build-essential \
+  libpq-dev \
+  python-dev \
+  libffi-dev \
+  curl
+
+RUN curl -sL https://deb.nodesource.com/setup_4.x | bash -
+
+RUN apt-get install -y nodejs
+
+WORKDIR /apps
+RUN git clone https://github.com/metabrainz/coverart_redirect.git
+
+RUN npm install -g less
+
+WORKDIR /apps/coverart_redirect
+RUN pip install -U cffi
```

Comment:
It's needed to prevent some errors in the subsequent `pip install -r requirements.txt`. IIRC something about a particular version of cffi is broken, but installing the latest version first gets around this.